packages/spark/tests/scrollYDirection.tests.js

```diff
@@ -0,0 +1,10 @@
+/* global document describe it */
+import { expect } from 'chai';
+import sinon from 'sinon';
+import scrollYDirection from '../utilities/scrollYDirection';
+
+describe('scrollYDirection tests', () => {
+  it('should return down if diff is greater than 0', () => {
+
```

Comment:
missing tests

Reply:
This was one of the tests I struggled with because there are no parameters being passed. Any direction or help would be great!